### PR TITLE
New desc

### PR DESCRIPTION
**Pull request by shrutiverma29**
description added

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```
Comment (jetersen) [inline .idea/.gitignore:2]: You need to also remove `.idea` from the PR changes.

File: lib/default-config.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const { SORT_BY, SORT_DIRECTIONS } = require('./sort-pull-requests')
 const DEFAULT_CONFIG = Object.freeze({
   'name-template': '',
   'tag-template': '',
-  'change-template': `* $TITLE (#$NUMBER) @$AUTHOR`,
+  'change-template': `* $TITLE (#$NUMBER) @$AUTHOR $DESC`,
```
Comment (jetersen) [inline lib/default-config.js:6]: Should not be added to the default, as I think this is optional.

File: test/fixtures/config/config-with-yaml-exception.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-change-template: - #$NUMBER '$TITLE' @$AUTHOR
+change-template: - #$NUMBER '$TITLE' @$AUTHOR '$DESC'
```
Comment (jetersen) [inline test/fixtures/config/config-with-yaml-exception.yml:1]: No need to add it here...

File: test/schema.test.js [hunk 4]
```diff
@@ -26,7 +26,10 @@ const invalidConfigs = [
   [{ template: { '👶': 'a' } }, 'must be a string'],
   [{ template: null }, 'must be a string'],
   [{ template: '' }, 'is not allowed to be empty'],
-  [{ 'change-template': ['* $TITLE (#$NUMBER) @$AUTHOR'] }, 'must be a string'],
+  [
+    { 'change-template': ['* $TITLE (#$NUMBER) @$AUTHOR $DESC'] },
```
Comment (jetersen) [inline test/schema.test.js:30]: add a new test

File: test/fixtures/config/config-with-resolved-version-template.yml [hunk 5]
```diff
@@ -11,7 +11,7 @@ categories:
     label: 'documentation'
   - title: '🧰 Maintenance'
     label: 'maintenance'
-change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
+change-template: '- $TITLE @$AUTHOR (#$NUMBER) $DESC'
```
Comment (jetersen) [inline test/fixtures/config/config-with-resolved-version-template.yml:14]: Should add a new test.

File: lib/default-config.js (hunk 2 above)
Comment (shrutiverma29) [reply lib/default-config.js:6]: Can you suggest an alternate way to add optional description in this template?

Comment (jetersen) [reply lib/default-config.js:6]: Simply remove it, and you inside your `release-drafter.yml` file you can override the `change-template` to include description.
Changing the default is a breaking change.

Comment (shrutiverma29) [reply lib/default-config.js:6]: But I want to add 'PR Description' which comes from api in body tag.

Comment (shrutiverma29) [reply lib/default-config.js:6]: @jetersen Oh yes, it's working. Thanks!

File: lib/commits.js [hunk 6]
```diff
@@ -33,6 +33,7 @@ module.exports.findCommitsWithAssociatedPullRequestsQuery = /* GraphQL */ `
                   nodes {
                     title
                     number
+                    body
```
Comment (jetersen) [inline lib/commits.js:36]: this

File: lib/releases.js [hunk 7]
```diff
@@ -144,7 +144,8 @@ const generateChangeLog = (mergedPullRequests, config) => {
         template(config['change-template'], {
           $TITLE: pullRequest.title,
           $NUMBER: pullRequest.number,
-          $AUTHOR: pullRequest.author ? pullRequest.author.login : 'ghost'
+          $AUTHOR: pullRequest.author ? pullRequest.author.login : 'ghost',
+          $DESC: pullRequest.body
```
Comment (jetersen) [inline lib/releases.js:148]: and this, is the only required change for the `$DESC` being available

Comment (shrutiverma29) [reply lib/releases.js:148]: Thanks @jetersen 
what's the process of making a github action so that I can use the updated code in my private repo?

**Pull request closed without merging**